### PR TITLE
Add release notes for PR 2631

### PR DESCRIPTION
**Pull request by sigmavirus24**
/cc @neosab @Lukasa

### CODE REVIEW

File: 3.0-HISTORY.rst [hunk 1]
```diff
@@ -8,4 +8,9 @@
   to most people and breaks most of the assumptions that people have about
   Response objects. This resolves issue `#2002`_
 
+- Skip over empty chunks in iterators. Empty chunks could prematurely signal 
+  the end of a request body's transmission, skipping them allows all of the 
+  data through. See `#2631` for more details.
```
Comment (Lukasa) [inline 3.0-HISTORY.rst]: Missing underscore. I'd fix it myself but I'm on my phone.

Comment (sigmavirus24) [inline 3.0-HISTORY.rst]: Woops. Fixed.

**Comment by Lukasa** [comment]
:cake: :sparkles:

**Comment by neosab** [comment]
Thanks @sigmavirus24

**Pull request merged**